test/browser/spec/ol/View.test.js

```diff
@@ -617,6 +617,19 @@ describe('ol/View', function () {
       expect(options.zoom).to.eql(10);
     });
 
+    it('returns the current properties with getProperties()', function () {
```

Comment:
Is this really going to return the "current" properties?  It seems like instead it will return a mix of the initial and the current properties.  Like the initial `zoom` and the current `resolution`.

Reply:
Good point! I changed things now so `zoom` is also updated as view property.